openday_scavenger/views/game/static/index.html

```diff
@@ -1,52 +1,105 @@
 {% extends "layout.html" %}
 
 {% block content %}
+<div class="container my-4 text-center">
 
+    <h1 class="type-h1 mb-3">🗝 Unlock the Secrets of the 🗝<br />🕵️‍♂️ Australian Synchrotron 🕵️‍♂️</h1>
+    <p>
+        Welcome to the Australian Synchrotron Open Day Scavenger Hunt!<br />
+        (Brought to you by Scientific Computing! 🧑‍💻)
+    </p>
+
+    {% if visitor.is_authenticated %}
+
+    {% if has_completed_all_puzzles %}
+
+    <p>
+        <span class="text-primary fw-bold">🎊 Congratulations!!</span><br />
+        You've completed all the puzzles!
+    </p>
+    <p>
+        Come back to the Scientific Computing booth and show us your journey key to pick up your prize! 🎖
+    </p>
+
+    {% else %}
+
+    <p class="mt-5">
+        <span class="text-primary">🔍 Find Puzzle Locks! 🔍</span><br />
+        Look for puzzle locks (QR codes marked for the Scavenger Hunt) around the Synchrotron!
+    </p>
+    <p>
+        Once you find a puzzle lock,<br />
+        scan it with your own phone camera or use our handy in-built scanner below!
+    </p>
+
+    <p class="text-primary fw-bold mt-5">Scan puzzle lock to unlock the puzzle!</p>
+    <div class="mx-auto" id="reader" style="width: 400px;"></div>
+
+    <div class="mt-3">
+        {% if number_correct_responses > 0 %}
+        <p class="text-primary">
+            Current # of correct answers:
+        </p>
+        <div class="progress" role="progressbar">
+            <div class="progress-bar" style="width: {{ (number_correct_responses / number_active_puzzles) * 100.0 }}%">
+                {{
+                number_correct_responses }} / {{ number_active_puzzles }}</div>
+        </div>
+        {% endif %}
+    </div>
+
+    <p class="mt-5">
+        <span class="text-primary">⚠️ Stay safe!!</span><br />
+        Please stay aware of your surroundings while playing,<br />
+        Puzzle locks will only be within accessible areas!
+    </p>
+
+    <p class="text-secondary mt-5">Need some help finding puzzle locks?<br />
+        <a href="/puzzles/map">Use our map to see where all the puzzle locks are!</a>
+    </p>
+
+    {% endif %}
+
+    {% else %}
+
+    <p>To join, come to the Scientific Computing booth to get your journey key! 🗝</p>
```

Comment:
Journey Key or Adventure Key @portrain @chanmSC

Reply:
Updated to adventure key!